lib/database/leaderboard.php

```diff
@@ -548,10 +549,21 @@ function GetLeaderboardData($lbID, $user, $numToFetch, $offset, $friendsOnly, $n
     return $retVal;
 }
 
+function _FormatValueSeconds($hours, $mins, $secs)
```

Comment:
As a general purpose string formatting function I think it'd be fitting to move it to `util/string.php` or similar.
Underscore prefixed functions in PHP are usually reserved for runtime/lang level functions - the non-prefixed camelCase variant (snake case even) would be better practice `formatValueSeconds` imo.
I'm aware that most of our functions are PascalCased, but let that slide because of the legacy character of the code base. Let's stick to the existing code style at least

Reply:
The intent was to make it a private function. Underscore is the standard way to do that in python. What is the PHP equivalent?

Reply:
Got it - technically it would be a private method within a class but we do not have a lot of classes in this codebase.
It's still common to have global functions in PHP which usually are camel cased - there's no way to declare it as private in this flat/function based style we have here.
I understand that we'll probably only ever need this function in the context of leaderboards, so it's fine to keep it this way if you prefer it. I'm personally used to mark private members and functions as private with underscores, it's just not common in this very codebase. Up to you - I'm fine with either, really.